panacea/sgxdb_util.go

```diff
@@ -0,0 +1,83 @@
+package panacea
+
+import (
+	"bytes"
+	"encoding/gob"
+	"github.com/medibloc/panacea-doracle/store/sgxleveldb"
+	log "github.com/sirupsen/logrus"
+	"os"
+	"path/filepath"
+)
+
+func SaveTrustedBlockInfo(info TrustedBlockInfo) error {
+	var buffer bytes.Buffer
+
+	enc := gob.NewEncoder(&buffer)
+
+	err := enc.Encode(info)
+	if err != nil {
+		return err
+	}
+
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		return err
+	}
+	dbDir := filepath.Join(userHomeDir, ".doracle", "data")
+
+	db, err := sgxleveldb.NewSgxLevelDB("light-client-db", dbDir)
+	if err != nil {
+		return err
+	}
+	err = db.Set([]byte("trustedBlockInfo"), buffer.Bytes())
+	if err != nil {
+		return err
+	}
+
+	err = db.Close()
+	if err != nil {
+		return err
+	}
```

Comment:
Why don't you call the `db.Close()` in defer function?

Reply:
Thanks, I fixed it in [c003735](https://github.com/medibloc/panacea-doracle/pull/12/commits/c00373570e183550673268dde1e2e71c7958e956).